cumulusci/tasks/bulkdata/generate_and_load_data.py

```diff
@@ -139,3 +186,22 @@ def _generate_batch(self, database_url, tempdir, mapping_file, batch_size, index
             if not subtask_options.get("mapping"):
                 subtask_options["mapping"] = generated_mapping_file.name
             self._dataload(subtask_options)
+
+    def _setup_engine(self, database_url):
+        """Find all tables that do not relate to id->OID mapping"""
```

Comment:
I don't think I understand this docstring for this method.

Reply:
Cut and paste error.

Reply:
974ba34cff42